Deploy-CAM.ps1

```diff
@@ -1172,8 +1192,15 @@ function New-ConnectionServiceDeployment() {
         $rg = Get-AzureRmResourceGroup -ResourceGroupName $csRGName -ErrorAction SilentlyContinue
         if($rg)
         {
-            # found the resource group - do the loop with an incremented number try to find a free name
-            $csRGName = $null
+            # Check if Resource Group is empty
+            $Resources = Find-AzureRmResource -ResourceGroupNameEquals $csRGName
+            $outputstr = $RG.ResourceGroupName + " - " + $Resources.Length
+            Write-Output $outputstr
```

Comment:
This a debug print? (Also note that Write-Output populates the object pipeline where Write-Host doesn't, I think.)

Reply:
Yes, not neccessary, will remove